pkg/registry/controller/rest.go

```diff
@@ -47,14 +48,26 @@ func (rcStrategy) NamespaceScoped() bool {
 func (rcStrategy) PrepareForCreate(obj runtime.Object) {
 	controller := obj.(*api.ReplicationController)
 	controller.Status = api.ReplicationControllerStatus{}
+	controller.Spec.SequenceNumber = 0
+	controller.Spec.SequenceNumber = 0
```

Comment:
Arguably this should not be here at all; it looks like defaulting, so it should go in defaulting code, but 0 is the default for a number anyway so you don't really need to do anything?